lib/Service/Appointments/BookingCalendarWriter.php

```diff
@@ -61,6 +63,22 @@ public function __construct(IConfig $config,
 		$this->random = $random;
 	}
 
+	private function secondsToIso8601Duration(int $secs): string {
```

Comment:
I would have simply added seconds to a timestamp, then DateTime::diff, then something like https://stackoverflow.com/a/61088115

But this is small too and works. :)

Reply:
There are so many solutions for this problem 🙈